### PR TITLE
Update urls in warnings for rich rendering

### PR DESCRIPTION
**Pull request by IvanReznikov**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by amyeroberts** [comment]
Hi @IvanReznikov - thanks for opening a PR. Could you expand a bit on the issue this is addressing? All I'm seeing in the diff is splitting the `")"` bracket onto a new line.

**Comment by IvanReznikov** [comment]
@amyeroberts,  the bracket was part of the URL, what leads to 404 obviously.

**Comment by amyeroberts** [comment]
@IvanReznikov The bracket is just the closing bracket the opened with `"(see..."` in the line above, it's not part of the url. Splitting like this won't change the evaluated string because of python's implicit line continuation behaviour.

**Comment by IvanReznikov** [comment]
image

**Comment by IvanReznikov** [comment]
@amyeroberts yep, fixed it above

File: src/transformers/generation/utils.py [hunk 1]
```diff
@@ -1260,7 +1260,8 @@ def generate(
                         "You have modified the pretrained model configuration to control generation. This is a"
                         " deprecated strategy to control generation and will be removed soon, in a future version."
                         " Please use a generation configuration file (see"
-                        " https://huggingface.co/docs/transformers/main_classes/text_generation)"
+                        " https://huggingface.co/docs/transformers/main_classes/text_generation"
+                        " )"
```
Comment (amyeroberts) [inline src/transformers/generation/utils.py:1264]:
```suggestion
                        " https://huggingface.co/docs/transformers/main_classes/text_generation )"
```

File: src/transformers/generation/tf_utils.py [hunk 2]
```diff
@@ -746,7 +746,8 @@ def generate(
                         "You have modified the pretrained model configuration to control generation. This is a"
                         " deprecated strategy to control generation and will be removed soon, in a future version."
                         " Please use a generation configuration file (see"
-                        " https://huggingface.co/docs/transformers/main_classes/text_generation)"
+                        " https://huggingface.co/docs/transformers/main_classes/text_generation"
+                        " )"
```
Comment (amyeroberts) [inline src/transformers/generation/tf_utils.py:750]:
```suggestion
                        " https://huggingface.co/docs/transformers/main_classes/text_generation )"
```

File: src/transformers/generation/flax_utils.py [hunk 3]
```diff
@@ -319,7 +319,8 @@ def generate(
                         "You have modified the pretrained model configuration to control generation. This is a"
                         " deprecated strategy to control generation and will be removed soon, in a future version."
                         " Please use a generation configuration file (see"
-                        " https://huggingface.co/docs/transformers/main_classes/text_generation)"
+                        " https://huggingface.co/docs/transformers/main_classes/text_generation"
+                        " )"
```
Comment (amyeroberts) [inline src/transformers/generation/flax_utils.py:323]:
```suggestion
                        " https://huggingface.co/docs/transformers/main_classes/text_generation )"
```

**Review by amyeroberts — COMMENTED** [review]
Thanks for adding! 

I've made some small suggestions to the change as there's no need to split on a new line

**Comment by IvanReznikov** [comment]
@amyeroberts , sure

**Review by amyeroberts — APPROVED** [review]
Thanks for iterating and adding this improvement!

**Pull request merged**